cmake/frontend/aardvark.cmake

```diff
@@ -1,13 +1,195 @@
-
 ################################################################################
 ## web interface build
 ################################################################################
-add_custom_target(frontend
+
+find_package(Nodejs 16 REQUIRED)
+find_package(Yarn REQUIRED)
+
+set(FRONTEND_SRC
```

Comment:
I know this looks weird, but is a "best practice" in CMake projects to explicitly list all dependencies; I am not sure whether it is just the `.js` files that constitute frontend sources or whether more files should be added here.

CMake's dependency tracking will trigger a rebuild of the frontend whenever a file in this list is touched, and cmake will complain if a file in this list is missing.

Of course it will not complain if there are excess files, i.e. a developer forgot to add a file here.

Reply:
TBH I don't think it makes sense to track all the js files in cmake. CMake only needs to create the `frontend` target for `make` which will then trigger `webpack` for the actual build which will automatically pick up all the required js files anyway.

So I think we should just remove this.

Reply:
@pluma4345 can we remove this as discussed? I think after that we are good to go for this